### PR TITLE
feat: add protos for Fine Grained Access Control

### PR DESCRIPTION
**Pull request by ansh0l**
Creating PR for FGAC protos.

### CODE REVIEW

**Comment by ansh0l** [comment]
@asthamohta Please review protos for FGAC.

File: google/spanner/admin/database/v1/backup.proto [hunk 1]
```diff
@@ -249,7 +249,7 @@ message CopyBackupRequest {
   CopyBackupEncryptionConfig encryption_config = 5 [(google.api.field_behavior) = OPTIONAL];
 }
 
-// Metadata type for the google.longrunning.Operation returned by
+// Metadata type for the operation returned by
```
Comment (asthamohta) [inline google/spanner/admin/database/v1/backup.proto:252]: Backup file should not be updated, can you remove this file?

**Comment by asthamohta** [comment]
Session proto should have changes for adding creator role as well. That is missing. Looks like that file was not added. Can we add that too in the same PR?

**Review by harshu1234567 — APPROVED** [review]

**Comment by jskeet** [comment]
Is this PR still required? Can we close it and delete the branch?

**Pull request closed without merging**